src/components/AssignTeamTableCell/index.tsx

```diff
@@ -0,0 +1,163 @@
+    stopPropagationAndPreventDefault(e);
+    setIsActive(!isActive);
+  };
+
+  const AssignTeamButton = (
+    <Button
+      color="primary"
+      id={getButtonId(jurisdictionId)}
+      onClick={onPlanAssignmentButtonClick}
+      size="sm"
+    >
+      {ASSIGN_TEAMS}
+    </Button>
+  );
+
+  const organizationsArray: Organization[] | null =
+    organizationsById &&
+    keys(organizationsById)
+      .map((o: string) => organizationsById[o])
+      .filter(o => !!o);
+
+  const organizationSelectProps = {
+    jurisdictionId,
+    name: getFormName(jurisdictionId),
+    planId,
+  };
+
+  const popoverProps = {
+    isOpen: isActive,
+    onClick: (e: React.MouseEvent) => stopPropagationAndPreventDefault(e),
+    target: getButtonId(jurisdictionId),
+    toggle: () => setIsActive(!isActive),
+  };
+
+  const AssignPopover = (
+    <Popover {...popoverProps}>
+      <PopoverHeader>Select Teams to Assign</PopoverHeader>
+      <PopoverBody>
+        {organizationsArray ? (
+          <Form name={getFormName(jurisdictionId)}>
+            <OrganizationSelect {...organizationSelectProps} />
+            <Button color="default" onClick={onClearAssignmentsButtonClick} size="xs">
+              Clear
+            </Button>
+            <Button color="primary" onClick={onSaveAssignmentsButtonClick} size="xs">
+              Save
+            </Button>
+          </Form>
+        ) : (
+          <p>No teams loaded...</p>
+        )}
+      </PopoverBody>
+    </Popover>
+  );
+
+  return (
+    <div onClick={stopPropagationAndPreventDefault}>
+      <span style={{ paddingRight: '2rem' }}>
+        {`${assignments.length} ${TEAMS_ASSIGNED}`}&nbsp;
+      </span>
+      {AssignTeamButton}
+      {AssignPopover}
+    </div>
+  );
+};
+
+const getButtonId = (jurisdictionId: string): string => `plan-assignment-${jurisdictionId}`;
+const getFormName = (jurisdictionId: string): string => `plan-assignment-form-${jurisdictionId}`;
+export { AssignTeamTableCell };
+
+/** map state to props
+ * @param {partial<store>} - the redux store
+ * @param {AssignTeamCellProps} ownProps - the props
+ *
+ * @returns {AssignTeamCellProps} - ownProps and organizationsById
+ */
+const mapStateToProps = (state: Partial<Store>, ownProps: any): AssignTeamCellProps => {
+  const organizationsById = getOrganizationsById(state);
+  const assignments = getAssignmentsArrayByPlanIdByJurisdictionId(
+    state,
+    ownProps.planId,
+    ownProps.jurisdictionId
+  );
+  return {
+    ...ownProps,
+    assignments,
+    organizationsById,
+  } as AssignTeamCellProps;
+};
+
+/** map props to actions that may be dispatched by component */
+const mapDispatchToProps = {
+  fetchAssignmentsActionCreator: fetchAssignments,
+};
+
+/** Create Connected AssignTeamTableCell */
+const ConnectedAssignTeamTableCell = connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(AssignTeamTableCell);
+
+export default ConnectedAssignTeamTableCell;
```

Comment:
We should move `AssignTeamTableCell` to the containers directory because it is more of a ... container